Tests/ReactiveSwiftTests/ActionSpec.swift

```diff
@@ -411,6 +411,21 @@ class ActionSpec: QuickSpec {
 
 				expect(values) == [3, 4, -11]
 			}
+			it("is disabled if the validating property does not hold a valid value") {
+				enum TestValidationError: Error { case generic }
+				typealias PropertyType = ValidatingProperty<Int, TestValidationError>
+				let decisions: [PropertyType.Decision] = [.valid, .invalid(.generic), .coerced(10, nil)]
+				let input = PropertyType(0, { decisions[$0] })
+				let action = Action(validating: input, execute: echo)
```

Comment:
Can you also test the other variant of this method?

Reply:
I simply copied the one from `unwrapping` and adjusted it to work with the `ValidatingProperty`. Is that ok?

Reply:
That works. 👍